scripts/pinecone_setup.py

```
@@ -0,0 +1,342 @@
+                    dimension = stats.get('dimension', 0)
+
+                    total_vectors += vectors
+                    total_dimensions += dimension
+
+                    print(f"📁 {idx_name}")
+                    print(f"   Vectors: {vectors:,}")
+                    print(f"   Dimension: {dimension}")
+                    print(f"   Storage: ~{(vectors * dimension * 4 / 1024 / 1024):.2f} MB")
+                    print()
+
+                except Exception as e:
```

Ruff (0.14.5)

Do not catch blind exception: `Exception`

(BLE001)